qase-python-commons/src/qase/commons/client/base_api_client.py

```diff
@@ -0,0 +1,89 @@
+import abc
+from typing import Dict, Union
+
+from qaseio import Project, AttachmentGet
+
+from ..models import Attachment
+
+
+class BaseApiClient(abc.ABC):
+    @abc.abstractmethod
+    def get_project(self, project_code: str) -> Union[Project, None]:
+        """
+        Method loads project from Qase TestOps by code and returns project data
```

Comment:
You can omit (remove) the "Method" part in each docstring, because it's obvious here.

```suggestion
        Load a project from Qase TestOps by code and return project data
```

Reply:
Fixed